tools/fetch-sponsors.js

```diff
@@ -227,7 +227,7 @@ async function fetchGitHubSponsors() {
     // return an array in the same format as Open Collective
     const sponsors = organization.sponsorshipsAsMaintainer.nodes
         .map(({ sponsor, tier }) => ({
-            name: sponsor.name,
+            name: sponsor.name || sponsor.login,
```

Comment:
Is there anyway we'd not have both login and name values? 
I remember fixing the same for url once? https://github.com/eslint/eslint.org/pull/314/files

Reply:
#314 this is for fixing URL only right?

Reply:
No I meant we have to fix the way I did if we thing sponsor.name or sponsor.login will be null? else this looks good to me

Reply:
ya current PR itself looks good to me.

Reply:
> Is there anyway we'd not have both login and name values? I remember fixing the same for url once? https://github.com/eslint/eslint.org/pull/314/files

Nah, `login` is our github username, so it can't be `null`.